connector/github/github.go

```diff
@@ -372,6 +376,81 @@ func (c *githubConnector) groupsForOrgs(ctx context.Context, client *http.Client
 	return groups, fmt.Errorf("github: user %q not in required orgs or teams", userName)
 }
 
+func (c *githubConnector) userGroups(ctx context.Context, client *http.Client) (groups []string, err error) {
+	orgs, err := c.userOrgs(ctx, client)
+	if err != nil {
+		return groups, err
+	}
+
+	orgTeams, err := c.userOrgTeams(ctx, client)
+	if err != nil {
+		return groups, err
+	}
+
+	for _, org := range orgs {
+		if teams, ok := orgTeams[org]; !ok {
+			groups = append(groups, org)
+		} else {
+			for _, team := range teams {
+				groups = append(groups, formatTeamName(org, team))
+			}
+		}
+	}
+
+	return groups, err
```

Comment:
[nit] `return groups, nil`, maybe? Just for clarity

Reply:
Agree. Fixed